### DISCUSSION TITLE
How to contibute to Maui

### DISCUSSION THREAD

**Discussion by HobDev** [dotnet/maui#3665] [Q&A]
Is there any guidance for new developers to start contributions to .NET maui. I didn't find any. I don't know the answer to even basic questions like can I contribute to maui with vscode or vs 2022 is required.  Some explanation of the repository's architecture will be very helpful.

**Comment by nogginbox** [comment]
Have you read the page on [how to contribute](https://github.com/dotnet/maui/blob/main/.github/CONTRIBUTING.md)?

This describes several ways that you can contribute, and there is a section on how to contribute code, but at this early stage in the project I think they expect you to figure out things for yourself and are probably more likely to accept bug fixes etc.

**Comment by HobDev** [comment]
If proper documentation were available it would have made the process lot easier.
